src/xmpp_wf/get/xmpp_iq_gameroom_get.c

```diff
@@ -104,6 +104,9 @@ static void xmpp_iq_gameroom_get_cb(const char *msg,
                     code,
                     custom_code);
 
+        if (a->cb)
+            a->cb(NULL, a->args);
+
```

Comment:
You might want to call the callback with NULL also at line ~165

Reply:
done